SimTracker/TrackAssociatorProducers/plugins/QuickTrackAssociatorByHitsImpl.cc

```diff
@@ -5,6 +5,13 @@
 
 #include "SimTracker/TrackerHitAssociation/interface/TrackerHitAssociator.h"
 
+#include "DataFormats/TrackerRecHit2D/interface/SiPixelRecHit.h"
```

Comment:
Where are all these added include's needed?
Please remove if not needed

Reply:
Apparently not, removed.